### PR TITLE
Create native "getting started" image without having to install GraalVM

### PR DESCRIPTION
**Pull request by SchulteMarkus**
"-Dquarkus.native.container-build=true" can be added to quarkus-maven-plugin for creating a native "getting started" image without the need to install GraalVM locally. Instead, docker image quay.io/quarkus/ubi-quarkus-native-image is used for as build image. 
See https://quarkus.io/guides/building-native-image#quarkus-native-pkg-native-config_quarkus.native.container-build for details

### CODE REVIEW

File: getting-started/README.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ Compiling a native executable takes a bit longer, as GraalVM performs additional
 steps to remove unnecessary codepaths. Use the  `native` profile to compile a
 native executable:
 
-> ./mvnw package -Dnative
+> ./mvnw package -Dnative -Dquarkus.native.container-build=true
```
Comment (cescoffier) [inline getting-started/README.md]: The flag is only required to build a container-compatible executable.

**Comment by gsmet** [comment]
We require GraalVM at the beginning of the README so I'm not sure about that.

In any case, the PR should target the `development` branch or it will get erased.

**Comment by SchulteMarkus** [comment]
Closing here as wrong base branch.

**Comment by SchulteMarkus** [comment]
Follow-up is #442

File: getting-started/README.md (hunk 1 above)
Comment (SchulteMarkus) [reply getting-started/README.md]: You seem to be wrong here. Omitting the flag requires ```native-image``` being present as executable locally. Comment please in follow-up #442

**Comment by SchulteMarkus** [comment]
> We require GraalVM at the beginning of the README so I'm not sure about that.
> 
> In any case, the PR should target the `development` branch or it will get erased.

PR recreated with correct base branch at #442

**Pull request closed without merging**